app/Models/Foundation/Main/Audit/AuditLog.php

```diff
@@ -0,0 +1,69 @@
+<?php namespace models\main;
+/**
+ * Copyright 2022 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+use Doctrine\ORM\Mapping AS ORM;
+use models\utils\SilverstripeBaseModel;
+
+/**
+ * @ORM\Entity(repositoryClass="repositories\main\DoctrineAuditLogRepository")
+ * @ORM\Table(name="AuditLog")
+ * @ORM\InheritanceType("JOINED")
+ * @ORM\DiscriminatorColumn(name="ClassName", type="string")
+ * @ORM\DiscriminatorMap({
+ *     "SummitAuditLog" = "SummitAuditLog",
+ *     "SummitEventAuditLog" = "SummitEventAuditLog",
+ * })
+ * Class AuditLog
+ * @package models\main
+ */
+abstract class AuditLog extends SilverstripeBaseModel
+{
+    /**
+     * @ORM\Column(name="User", type="string")
+     * @var string|null
+     */
+    private $user;
```

Comment:
user should be a Member not a string should point to member table ( FK)

Reply:
done